### PR TITLE
fix(app): fix lastRun on RecentRunProtocolCard

### PR DESCRIPTION
**Pull request by koji**
# Overview
current code displays the oldest createdAt time instead of the latest time. so added sort before passing data to RecentRunProtocolCard.

[the fixed image](https://opentrons.atlassian.net/browse/RCORE-762?focusedCommentId=80992)

close RCORE-762

# Test Plan

# Changelog
- add sort to RecentRunProtocolCarousel

# Review requests

# Risk assessment
low

### CODE REVIEW

**Review by jerader — APPROVED** [review]
lgtm!

**Comment by koji** [comment]
- [ ] merge the test fix PR

File: app/src/organisms/OnDeviceDisplay/RobotDashboard/RecentRunProtocolCarousel.tsx [hunk 1]
```diff
@@ -69,9 +69,13 @@ export function RecentRunProtocolCarousel({
     >
       <Flex flexDirection={DIRECTION_ROW} gridGap={SPACING.spacing8}>
         {sortedProtocols.map((protocol: ProtocolResource) => {
-          const run = runs.data?.data.find(
-            run => run.protocolId === protocol.id
-          )
+          const run = runs.data?.data
```
Comment (b-cooper) [inline app/src/organisms/OnDeviceDisplay/RobotDashboard/RecentRunProtocolCarousel.tsx]: How is this different than the "recentRun" `sortBy` option for the `sortProtocols` helper function?  It seems like that function should be responsible for this sorting right?

Comment (koji) [reply app/src/organisms/OnDeviceDisplay/RobotDashboard/RecentRunProtocolCarousel.tsx:72]: I will check this when I'm in the office.

**Comment by koji** [comment]
- [ ] need to re-review the sorting function

**Comment by koji** [comment]
`sortProtocols` function has a bug.
- [x] fix the sorting bug

**Comment by koji** [comment]
This issue will be fixed by a following PR

**Pull request closed without merging**